config/deploy.rb

```diff
@@ -20,10 +20,10 @@
 # set :pty, true
 
 # Default value for :linked_files is []
-set :linked_files, fetch(:linked_files, []).push('config/database.yml', 'config/secrets.yml', 'public/.htaccess', 'config/honeybadger.yml')
+set :linked_files, %w['config/database.yml', 'config/secrets.yml', 'public/.htaccess', 'config/honeybadger.yml']
```

Comment:
@camillevilla retaining the single quotes while using `%w` keeps the `'` in the value (which we don't want.

`%w['config/database.yml', 'config/secretes.yml']`
yields
`["'config/database.yml',", "'config/secretes.yml'"]`

while 

`%w[config/database.yml config/secretes.yml]`
yields
`["config/database.yml", "config/secretes.yml"]`